packages/template-ui/src/views/Test.vue

```diff
@@ -0,0 +1,85 @@
+<template>
```

Comment:
This is a nice way to show the custom channel theme. Maybe we can add a link to this in the `DevTag` for each custom channel.

Reply:
That's a great idea! It will allow Joana to see it.